blog-posts/the-modern-code-reviewer-2021.md

```diff
@@ -0,0 +1,219 @@
+
+# The solution
+
+Did you see yourself or your former self in some of the described situations? We are humans and not perfect. It simply happens that we procrastinate, that we are captured [in the zone](https://en.wikipedia.org/wiki/Flow_(psychology)) (for better or worse) or intimidated by changes. Instead of changing people we can offer a better approach to positively influence their habits. That's where I am keen to introduce different code review practices.
+
+If you find your team regularly discussing codes style issues like [code indentations](https://www.youtube.com/watch?v=SsoOG6ZeyUI), [semicolons](https://github.com/twbs/bootstrap/issues/3057) or [curlies](https://softwareengineering.stackexchange.com/questions/2715/should-curly-braces-appear-on-their-own-line) you clearly missed to define a common code style through a contribution guideline. You are wasting precious review time. Fix this before optimizing anything else beyond this point.
+
+There is a total of three code review approaches to help your team improving their reviewing throughput. We introduce the **shallow review**, the **common review** and the **expert review** which primarily differ in the reviewing intensity. The more intense the higher the amount of time you have to invest. Your average code review probably corresponds to the common review with some reviews being of the expert format.
+
+The novelty of the approach is the explicitly given choice. A contribution guideline rarely describes how to conduct a review so people usually conduct code reviews following their gut feeling and to fit expectations. This means it highly depends on the person how a review is conducted. You are the alpha geek of your team? Of course you have to go into depth for every single PR which will make you a bottle neck if everyone relies on your reviews. Your are a less experienced developer? You will probably conduct a common review which takes high amounts of time for you, because of the overwhelming complexity or size of your project. You are new to the team and you fear to provide a common review so you decide not to take part in the review process at all.
+
+This code review pattern gives you a choice so you can fit more reviews in the same amount of time while dedicating enough time to the important changes while reducing your investment on trivial changes or things that can easily be changed if not perfect.
+
+## The shallow review
+
+T**he shallow review** is best ****suited for peers who are in a hurry (lacking time), not deeply involved (lacking knowledge) or feel intimidated by the code or experience of the reviewee (lacking confidence). A majority of pull requests should be suitable for this code review type, as long and they have a small enough size. When a code change introduces major architectural changes, people with shallow reviews are still welcomed but the reviewee must additionally require in-depth reviews.
```

Comment:
probably the stars got a bit mixed up in this markup ✨ 
also a typo in "as long **as** they have a small enough size" and possibly "with shallow reviews are still **welcome** but"